nautobot/docs/configuration/required-settings.md

```diff
@@ -2,7 +2,7 @@
 
 ## ALLOWED_HOSTS
 
-Environment Variable: `NAUTOBOT_ALLOWED_HOSTS` specified as a comma separated list.
+Environment Variable: `NAUTOBOT_ALLOWED_HOSTS` specified as a space-separated quoted string (e.g. `NAUTOBOT_ALLOWED_HOSTS="localhost 127.0.01 example.com"`).
```

Comment:
```suggestion
Environment Variable: `NAUTOBOT_ALLOWED_HOSTS` specified as a space-separated quoted string (e.g. `NAUTOBOT_ALLOWED_HOSTS="localhost 127.0.0.1 example.com"`).
```

What was the driver for the change from comma-separated to space-separated?

Reply:
It's in the commit message, but I just updated the PR description to have it. Seeing as it wasn't working before, and this is new as of v1.0.0:

> Additionally to bring it into parity w/ the development config (which has been around longer), the NAUTOBOT_ALLOWED_HOSTS environment variable now splits on a SPACE, not a comma.

It should be one or the other and since the development config has been around longer, and many people are using it, I decided it would be best to align around spaces. Commas are fine, too, but we should have the same pattern in all places.